src/main/java/com/google/devtools/build/lib/remote/SimpleBlobStoreFactory.java

```diff
@@ -68,8 +76,16 @@ public static SimpleBlobStore create(
             + " options expected.");
   }
 
-  public static boolean isRemoteCacheOptions(RemoteOptions options) {
-    return isHttpUrlOptions(options) || isDiskCache(options);
+  // TODO: This function can be expanded to combine disk cache and gRPC cache
```

Comment:
add here an issue id for the future reference/

Reply:
Done.